building-epics/getting-epics7-on-rtems5-beatnik.rst

```diff
@@ -0,0 +1,4365 @@
+Getting EPICS 7 on RTEMS 5 (beatnik VMEbus board)
```

Comment:
Maybe we should create a section "Latest developments" or "News"?
I can relate to all of Andrew's comments. My point was to support the current "hype" around RTEMS5. I wanted to give a documentation as accurate as possible. E.g. I used the IP addresses so that you can find the same server IP at beatnik-boot etc. I'm happy to modify and customize it as Andrew suggested and wait until we have an "official" github URL. I have already reached the currently active players ;-)
To the environments :
export RTEMS_MAKEFILE_PATH=${RTEMS_ROOT}/${RTEMS_ARCH}/${RTEMS_BSP}
export RTEMS_SHARE_PATH=${RTEMS_ROOT}/share/rtems${RTEMS_VERSION}
These are expected from the libbspext - Makefile.

Reply:
"latest" or "news" creates a maintenance task, as items have to be removed after they lose their novelty aspect.
We do have a "news" section on the main site - that's where announcements go.

How-Tos are instructions. As such they should be comprehensive and correct. A user following them doesn't care how old they are, as long as they work.